workers/java/worker/src/main/java/com/hrznstudio/spatial/worker/entity/EntityWorker.java

```diff
@@ -42,6 +44,11 @@ protected void onConnected() {
             }
         });
 
+        getDispatcher().onCommandRequest(PlayerInput.Commands.ITEM_RIGHTCLICK, argument -> {
+            System.out.println("Received right click with hand " + (argument.request.getMain() ? EnumHand.MAIN_HAND : EnumHand.OFF_HAND).name());
```

Comment:
Please use a logger !

Reply:
its a temporary debug output

Reply:
I've seen those temporary debug outputs too many times in production :/